climpred/prediction.py

```diff
@@ -125,171 +128,54 @@ def compute_reference(ds,
     metric (str):
         Metric used in comparing the decadal prediction ensemble with the
         reference.
-        * pearson_r (Default)
-        * rmse
-        * mae
-        * mse
     comparison (str):
         How to compare the decadal prediction ensemble to the reference.
         * e2r : ensemble mean to reference (Default)
         * m2r : each member to the reference
     nlags (int): How many lags to compute skill/potential predictability out
                  to. Default: length of `lead` dim
-    return_p (bool): If True, return p values associated with pearson r.
 
     Returns:
         skill (xarray object): Predictability with main dimension `lag`.
-        p_value (xarray object): If `return_p`, p values associated with
-                                 pearson r correlations.
     """
+    check_xarray(hind)
```

Comment:
This shouldn't be here and might be a rebasing mess-up. See the master branch where @ahuang11 implemented the `check_xarray` decorator. Can convert these back to decorators.